### PR TITLE
feat: allow cartservice to accept `POSTGRES` env var

### PR DESCRIPTION
**Pull request by tedim52**

### CODE REVIEW

File: src/cartservice/cartstore/db.go [hunk 1]
```diff
@@ -49,7 +59,17 @@ func retryConnect(dsn string, maxRetries int, initialBackoff time.Duration, back
 	backoff := initialBackoff
 
 	for i := 0; i < maxRetries; i++ {
-		// Attempt to execute the operation
+		// Need to change the resolver to resolve all addresses to use ipv4 instead of ipv6
+		net.DefaultResolver = &net.Resolver{
```
Comment (leoporoli) [inline src/cartservice/cartstore/db.go:63]: wow! I didn't know this, how did you realize?

File: src/cartservice/Dockerfile [hunk 2]
```diff
@@ -18,8 +14,20 @@ COPY . .
...
-# Deployment container
-FROM scratch
+# # Deployment container
+FROM alpine:latest
+
+# Install dependencies
+RUN apk --update --no-cache add ca-certificates make protoc
+
+# These tools are for debuggin the containres
+RUN apk update && apk add --no-cache \
```
Comment (leoporoli) [inline src/cartservice/Dockerfile:24]: is it worth it to rerun the update? wasn't it executed in the previous line? I believe that you could merge these two.

**Review by leoporoli — APPROVED** [review]

File: src/cartservice/cartstore/db.go (hunk 1 above)
Comment (leoporoli) [reply src/cartservice/cartstore/db.go:63]: nice!

File: src/cartservice/Dockerfile (hunk 2 above)
Comment (tedim52) [reply src/cartservice/Dockerfile]: good catch

**Pull request merged**